net/pfSense-pkg-Quagga_OSPF/files/usr/local/pkg/quagga_ospfd.inc

```diff
@@ -294,25 +294,26 @@ function quagga_ospfd_install_conf() {
 	    && !empty($config['installedpackages']['quaggaospfdraw']['config'][0]['bgpd'])) {
 	    // if there is a raw config specified in the config.xml use that instead of the assisted config
 	    $bgpdconffile = str_replace("\r","",base64_decode($config['installedpackages']['quaggaospfdraw']['config'][0]['bgpd']));
-	} else {
-	    $bgpdconffile = "Use vtysh to modify content\n\n";
 	}
+	else 
```

Comment:
You should keep the braces here. See https://doc.pfsense.org/index.php/Developer_Style_Guide#Indent_style